src/App.css

```diff
@@ -74,3 +74,10 @@ label {
   flex-direction: column;
   align-items: center;
 }
+
+.delete-button {
+  color: red;
+  margin-left: 10px;
```

Comment:
You might be able to remove this line since line 82 sets all margins to 10 px?

Reply:
Thanks for catching this Rob! Removed the margin-left.